validator/sawtooth_validator/state/client_handlers.py

```diff
@@ -1094,3 +1102,10 @@ def _respond(self, request):
         return self._wrap_response(
             endpoint=self._gossip.endpoint,
             peers=sorted(peers, key=lambda peer: peer.endpoint))
+
+
+def client_batch_submit_request_preprocessor(message_content_bytes):
+    request = client_batch_submit_pb2.ClientBatchSubmitRequest()
+    request.ParseFromString(message_content_bytes)
```

Comment:
This still needs to correctly handle deserialization errors and respond back the client if that happens. This will require modification to the preprocessor callback (https://github.com/hyperledger/sawtooth-core/pull/1489/files#diff-5b210e75615ded71391b5b742dbc779eR202) to do something like: https://github.com/hyperledger/sawtooth-core/blob/master/validator/sawtooth_validator/networking/dispatch.py#L190